README.md

```diff
@@ -258,6 +258,15 @@
         <span>iCue</span>
       </p>
     </td>   
+    <td valign="top">
+      <p align="center">
+        <a href="https://github.com/daltonmenezes/aura-theme/tree/main/packages/plasma">
+          <img src="https://kde.org/stuff/clipart/logo/plasma-logo-colorful-128x128.png" align="center" height="96px" />
```

Comment:
Update it please to `https://github.com/daltonmenezes/assets/blob/master/images/icons/plasma.png?raw=true`